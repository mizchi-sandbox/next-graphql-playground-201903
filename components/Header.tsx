import React from "react";
import Link from "next/link";

export function Header() {
  return (
    <header>
      <h1>Next Playground</h1>
      <div>
        <Link href="/">
          <a>Index</a>
        </Link>
        |
        <Link href="/about">
          <a>About</a>
        </Link>
        |
        <Link href="/apollo">
          <a>Apollo</a>
        </Link>
      </div>
    </header>
  );
}
