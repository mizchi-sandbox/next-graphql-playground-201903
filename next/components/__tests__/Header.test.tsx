import React from "react";
import Renderer from "react-test-renderer";
import { Header } from "../Header";

test("snapshot example", () => {
  const tree = Renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
