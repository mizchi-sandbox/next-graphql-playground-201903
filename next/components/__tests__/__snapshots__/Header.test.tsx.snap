// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot example 1`] = `
<header>
  <h1>
    Next Playground
  </h1>
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      Index
    </a>
    |
    <a
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
    |
    <a
      href="/apollo"
      onClick={[Function]}
    >
      Apollo
    </a>
  </div>
</header>
`;
